### ISSUE TITLE
`<C-v>` raises an error in other buffers than neo-minimap

### ISSUE THREAD

**Issue by woosaaahh** [ziontee113/neo-minimap#11]
Hi !

With the latest commits from `master`, using `<C-v>` raises the error below after opening/closing the `neo-minimap` window at least once.
```
E5108: Error executing lua: ...e/pack/github/start/neo-minimap/lua/neo-minimap/init.lua:233: Invalid window id: 1006
stack traceback:
	[C]: in function 'nvim_win_call'
	...e/pack/github/start/neo-minimap/lua/neo-minimap/init.lua:233: in function <...e/pack/github/start/neo-minimap/lua/neo-minimap/init.lua:231>
```

Steps to reproduce : 
- Open any file that can be used with `neo-minimap`
- Open the `neo-minimap` window
- Close the `neo-minimap` window
- Make a block selection with `<C-v>`

From what I understand it seems that when the `neo-minimap` window is closed, the autocommands/keymaps are not "deleted".

**Comment by ziontee113** [comment]
Hi @woosaaahh , this is a bug, which I will provide a fix shortly.

**Comment by ziontee113** [comment]
Hi @woosaaahh , this bug should now be resolved at 50cce453b09dd0e57462b70aff5e9748532a380b. I too have the same problem :sweat_smile:
